### ISSUE TITLE
Two stage telemetry alarms

### ISSUE THREAD

**Issue by Tygrys-1** [DeviationTX/deviation#425]
As for the aircraft battery voltage I'd lika the two stage warnings:
1) battery is low
2) battery i s critical.
Can it be achived with the Deviationtx?  I know I can set whatever alarm on telemerty data, but i have only one stage of it.

**Comment by TheRealMoeder** [comment]
Use two telemetry alarms with the same parameter but other cutoff value?

**Comment by Tygrys-1** [comment]
Yes.

**Comment by howard0su** [comment]
We can make sure one data point change will only trigger one alarm. And the order of alarm is the priotiy of alarm. When critial trigger fired, we can stop the further processing. Does this sound reasonable?

**Comment by Tygrys-1** [comment]
Yes, this sounds good. Stop further processing of the sensor when meet the condition for this sensor.
Can you also look at the #440? It is somehow related.
